source/reference/api/start.txt

```diff
@@ -148,6 +150,10 @@ Request Body Parameters
 
        .. include:: /includes/api/facts/namespace-explanation.rst
 
+       The ``/start`` command returns an error when the
+       namespace filtering is enabled and
+       ``verification.enabled`` is set to ``true``.
```

Comment:
I read this to mean that namespace filtering isn't compatible with embedded verification, but it should be compatible - can we clarify further what this means?

Reply:
Fixed